index.js

```diff
@@ -0,0 +1,279 @@
+     * @returns expression with converted log function
+     */
+    function logappend(str){
+        var i = str.indexOf("LOG");
+        var j = i + 3;
+        var k = j;
+        var cma = -1;
+        var eq = false;
+        var open = 0;
+        var close = 0;
+    
+        while(!eq) {
+            if (str[k] == "(") {
+                open += 1;
+            }
+            if (str[k] == ")") {
+                close += 1;
+            }
+    
+            if (open == (close + 1) && k != j && str[k] == ","){
+                cma = k;
+            }
+            if (open == close) {
+                eq = true;
+            }
+            else {
+                k += 1;
+            }
+        }
+    
+        return str.slice(0, i).trim() + str.slice(cma + 1, k).trim() + ".log(" + str.slice(j+1, cma).trim() + ")" + str.slice(k+1).trim();
+    }
+    
+    /**
+     * Identifies expression to left of operator
+     * @param {string} str - inputted expression
+     * @param {int} i - operator index
+     * @returns expression to left of operator
+     */
+    function lfind(str, i) {
+        if (str[i-1] != ")") {
+            end = true;
+            lstr = "";
+            while(end) {
+                if(str[i-2] == undefined){
+                    end = false; 
+                }
+                if((/[a-z]|[0-9]|[.]|[-]/).test(str[i-1])) {
+                    lstr = str[i-1] + lstr;
+                    i -= 1;
+                }
+                else {
+                    end = false;
+                }
+            }
+            return lstr;
+        }
+        else {
+            var eq = false;
+            var open = 0;
+            var close = 0;
+            var k = i-1;
+        
+            while(!eq) {
+                if (str[k] == "(") {
+                    open += 1;
+                }
+                if (str[k] == ")") {
+                    close += 1;
+                }
+                if (open == close) {
+                    eq = true;
+                }
+                else {
+                    k -= 1;
+                }
+            }
+            return str.slice(k, i);
+        }
+    }
+    
+    /**
+     * Identifies expression to right of operator
+     * @param {string} str - inputted expression
+     * @param {int} i - operator index
+     * @returns expression to right of operator
+     */
+    function rfind(str, i) {
+        if (str[i+1] != "(") {
+            end = true;
+            rstr = "";
+            while(end) {
+                if(str[i+2] == undefined){
+                    end = false; 
+                }
+                if((/[a-z]|[0-9]|[.]|[-]/).test(str[i+1])) {
+                    rstr = rstr + str[i+1];
+                    i += 1;
+                }
+                else {
+                    end = false;
+                }
+            }
+            return rstr;
+        }
+        else {
+            return str.slice(i+1, str.slice(i).indexOf(")")+i+1);
```

Comment:
It shouldn't have an effect because the converter still keeps the same format. It would evaluate the outer parenthesis first but leave the inner one in the original form to evaluate it after.